src/javascripts/ng-admin/Crud/column/maColumn.js

```diff
@@ -36,7 +36,9 @@ function maColumn($state, $anchorScroll, $compile, Configuration, FieldViewConfi
             scope.datastore = scope.datastore();
             scope.field = scope.field();
             scope.entry = scope.entry();
-            scope.value = scope.entry.values[scope.field.name()];
+            scope.value = typeof scope.entry === 'undefined'
+                ? ''
+                : scope.entry.values[scope.field.name()];
```

Comment:
as for relying on truethiness vs. checking typeof, this could turn into a holy war which is far from my intent, so whichever is consistent with the project's guidelines is fine by me.